snooty/postprocess.py

```diff
@@ -976,7 +983,59 @@ def enter_node(self, fileid_stack: FileIdStack, node: n.Node) -> None:
             assert isinstance(argument, n.Reference)
             source = argument.refuri
 
-        self.openapi_pages[current_slug] = self.SourceData(source_type, source)
+        api_version = node.options.get("api_version", None)
+        resource_versions: Optional[List[str]] = None
+
+        # Fetch OpenAPI versioning data if options are present
+        if api_version and source == "cloud":
+            # Fetch latest git_hash for S3 versioning data
+            git_hash_response = requests.get("https://cloud.mongodb.com/version")
```

Comment:
There are two urls needed for this process. I am totally unaware of where to place these constants... I searched for other examples in the parser and really struck out. I believe most urls are passed in, maybe. Halp?

Reply:
Should we create a new directory to store constants? it could be like `snooty/constants/urls.py` maybe

Reply:
A new directory or file could work. @i80and, do you have any strong opinions here?

Reply:
I wouldn't be opposed to a `snooty/constants.py` file. Ideally anything like this should be configurable in `snooty.toml` for theoretical outside users, but this entire functionality sounds pretty mongodb-specific.

Either way, no need to do that in this PR.